bigchaindb/tendermint/event_stream.py

```diff
@@ -50,7 +50,7 @@ def process_event(event_queue, event, stream_id):
         # Only push non empty blocks
         if block_txs:
             block_txs = [decode_transaction_base64(txn) for txn in block_txs]
-            new_block = {'id': str(block_id), 'transactions': block_txs}
+            new_block = {'id': block_id, 'transactions': block_txs}
```

Comment:
I just noticed an inconsistency between this and what the docs say is inside a block. The [docs about **The Block Model**](https://docs.bigchaindb.com/projects/server/en/master/data-models/block-model.html) say that the two top-level keys in a block are `"height"` and `"transactions"` but here it's building a block with `"id"` and `"transactions"` as the top-level keys.

This inconsistency might be elsewhere as well.

Reply:
Maybe we could standardize on using `block_height` and "block height" everywhere.

Reply:
I created issue #2168 about the inconsistent key name.